package/cfg/k3s-cis-1.24-hardened/node.yaml

```
@@ -38,7 +38,7 @@ groups:
 
       - id: 4.1.3
         text: "If proxy kubeconfig file exists ensure permissions are set to 600 or more restrictive (Manual)"
-        audit: 'stat -c %a /var/lib/rancher/k3s/agent/kubeproxy.kubeconfig'
+        audit: 'stat -c permissions=%a /var/lib/rancher/k3s/agent/kubeproxy.kubeconfig'
```

```suggestion
        audit: '/bin/sh -c ''if test -e $proxykubeconfig; then stat -c permissions=%a $proxykubeconfig; fi'' '
```
This should work out of the box since it's already referenced in the [local config.yaml](https://github.com/rancher/security-scan/blob/dc3b15e6bcda50f08c69c01ffa93a6c67f4f72d3/package/cfg/k3s-cis-1.24-hardened/config.yaml#L42).